src/tests/spec/modal/modal.spec.js

```diff
@@ -73,10 +69,6 @@ describe('Component: Modal', function() {
         expect(document.querySelector('.ons-page').getAttribute('aria-hidden')).to.equal('false');
```

Comment:
Need to remove these aria-hidden tests also

Reply:
Done

Reply:
Decided to reinstate this as I think we still need to set aria-hidden values on the page and the modal